src/pages/Todo/Edit/TodoEdit.js

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Helmet } from 'react-helmet';
 
-export default function TodoEdit() {
-  return <div></div>;
+import { useGet } from '../../../hooks/useRequest';
+import { Spinner } from '@ui5/webcomponents-react';
+import NavBack from '../../../components/NavBack/NavBack';
+import CenteredContent from '../../../components/Layout/CenteredContent';
+import TodoEditForm from './TodoEditForm';
+import Constants from '../../../util/Constants';
+import APIProvider from '../../../util/api/url/APIProvider';
+
+export default function TodoEdit({ match }) {
+  const { t } = useTranslation();
+
+  const { data, status } = useGet(Constants.REACT_QUERY.KEYS.GET_TODO_BY_ID, APIProvider.getUrl('GET_TODO_BY_ID', [{ value: match.params.id }]));
+
+  const onSubmitEditForm = (values, actions) => {
```

Comment:
Every time there is a rerender this function will be created again. Since it does not depend on anything related to the component (state, props) it's better to declare it outside the component.

```js
// imports

const onSubmitEditForm = (values, actions) => {
// function code
}

export default function TodoEdit({ match }) {
// component code
}
```